ui/src/App.tsx

```diff
@@ -73,6 +70,11 @@ function mainComponent(connection: Connection, state: types.State, dispatch: Dis
   />
 }
 
+const disconnectedComponent = () => <div className="App_warning">
+  <div>Reconnecting...</div>
+  <div className="indication">The connection has been lost, you may not be uptodate.</div>
```

Comment:
I think "up to date" is in three words ;)

Reply:
👍